assets/parquet/main.py

```
@@ -0,0 +1,154 @@
+"""
+
+PARQUET_DATA_PATH = "/home/hive-data"
+PARQUET_SCHEMA_NAME = "bench_schema"
+PARQUET_TABLE_NAME = "bench_table"
+class parquet_bench(Benchmark):
+    # add any parameters to the tool here
+    def __init__(self, dataset, mode='json string'):
+        super().__init__(dataset)
+
+        if mode not in ('json string', 'pairwise arrays'):
+            raise Exception('mode must be either "json string" or "pairwise arrays"')
```

Ruff (0.12.2)

Create your own exception

(TRY002)